source/our_tools/tractoflow.rst

```diff
@@ -3,4 +3,220 @@
 Tractoflow
 ==========
 
-Coming soon
\ No newline at end of file
+.. role:: bash(code)
+   :language: bash
+
+Our lab has prepared pipelines as nextflows (see the 'Nexflow' section in 'Other tools'). You can analyse all your diffusion data in only one simple script that will run a pipeline called the `Tractoflow <https://tractoflow-documentation.readthedocs.io/en/latest/pipeline/steps.html>`_, which does all the preprocessing on DWI data and T1 data and performs the wholebrain tracking.
+
+Installing Tractoflow
+*********************
+
+    You can check the `installation guide <https://tractoflow-documentation.readthedocs.io/en/latest/installation/before_install.html>`_ on Tractoflow's website (pages before install and install). If you want to install it on Compute Canada, connect first and follow instructions in the High Performance Computer sections.
+
+Using Tractoflow on Compute Canada
+**********************************
+
+    To learn about Compute Canada, please see the :ref:`ref_heavy_computing` page.
+
+    To use tractoflow on a computing platform, you will find some information on Tractoflow's website, section `How to launch tractoflow <https://tractoflow-documentation.readthedocs.io/en/latest/pipeline/launch.html>`_. Here is additional, more complete instructions:
+
+    #. Bring data to your cluster. Ex, here, on Beluga.
+
+        .. code-block:: bash
+
+            scp MY_DATA.zip USER@beluga.computecanada.ca:
+            scp -r MY_DIR USER@beluga.computecanada.ca:  # To send a whole folder, unzipped.
+
+    #. Connect to the cluster using ssh. Type ls and make sure that you see
+
+        | tractoflow-x.x.x/
+        | tractoflow_x.x.x_version_20xx-xx-xx.img
+        | MY_DATA.zip
+
+    #. Bring your data to your scratch directory.
+
+        .. code-block:: bash
+
+            mkdir scratch/Project_X
+            mv MY_DATA.zip scratch/Project_X/
+            cd scratch/Project_X/
+            unzip MY_DATA.zip
+            rm -rf MY_DATA.zip
+
+    #. You now have all your input data ready to go for tractoflow. If you want to double-check that all the necessary files are there. Run
+
+        .. code-block:: bash
+
+            for a in input_data_tractoflow/*; do ls $a; done
+
+       You should see bval  bvec  dwi.nii.gz  t1.nii.gz, as many times as you have subjects.
+
+    #. Nextflow will need to find your data in the scratch. As explained on Tractoflow's website (section How to launch your data), create a file beluga.conf in your project (ex, using nano), which contains these 2 lines.
```

Comment:
This config file will be useless in the next release. TractoFlow will do the bind automatically. Normally the next release will be out this week.